great_expectations/cli/cli.py

```diff
@@ -299,18 +299,19 @@ def build_documentation():
 @click.option('--data_asset_name', '-dan',
               help='The data asset for which to generate documentation. Must also specify --site_name.')
 def build_docs(directory, site_name, data_asset_name):
-    """Build data documentation for a project."""
+    """Build Data Docs for a project. for a project."""
     logger.debug("Starting cli.build_docs")
+    cli_message("Building <green>Data Docs</green>!")
 
     if data_asset_name is not None and site_name is None:
-        cli_message("Error: When specifying data_asset_name, must also specify site_name.")
-        return
+        cli_message("<red>Error: When specifying data_asset_name, must also specify site_name.</red>")
```

Comment:
this feels like it's (still, I know the problem didn't come from here) missing a word. maybe "When specifying data_asset_name, site_name is required"

Reply:
Don't apologize. PRs are a great opportunity to spot things like this.